docs/api-docs/analytics/bodl-overview.md

```diff
@@ -0,0 +1,427 @@
+    event.preventDefault();
+    const formData = new FormData(event.target);
+    let productId, productQty;
+    for (const pair of formData.entries()) {
+      if (pair[0] === 'product_id') {
+        productId = pair[1];
+      } else if (pair[0] === 'qty[]') {
+        productQty = pair[1];
+      }
+    }
+
+    // References:
+    // https://developer.bigcommerce.com/theme-objects/product
+    // https://developer.bigcommerce.com/stencil-docs/developing-further/catalog-price-object
+    sampleAnalyticsProvider.instance('<%= property_id %>').track('AddToCart', {
+      content_id: BODL.product.id,
+      content_category: BODL.breadcrumbs[1] ? BODL.breadcrumbs[1].name : '',
+      content_type: 'product_group',
+      content_name: BODL.product.title,
+      quantity: productQty,
+      price: BODL.product.price.without_tax.value,
+      value: (BODL.product.price.without_tax.value * productQty),
+      currency: BODL.product.price.without_tax.currency,
+    });
+    
+  }));
+
+  if (BODL.cartItemAdded) {
+    sampleAnalyticsProvider.instance('<%= property_id %>').track('AddToCart', {
+      content_id: BODL.cartItemAdded.product_id,
+      content_type: 'product_group',
+      content_name: BODL.cartItemAdded.name,
+      quantity: BODL.cartItemAdded.quantity,
+      price: BODL.cartItemAdded.price.value,
+      value: BODL.cartItemAdded.total.value,
+      currency: BODL.cartItemAdded.price.currency,
+    });
+  }
+</script>
+```
+
+## Add to wishlist
+The following snippet handles the Add to Wishlist event. This snippet uses the `added_product_id` to capture data about the individual products a shopper adds to their wishlist. 
+
+```handlebars title="Sample Script Code Start: Add to Wishlist" lineNumbers
+<script>
+  // This only sends one wishlist product: the one that was just added based on the 'added_product_id' param in the url
+  if (BODL.wishlist) {
+    var addedWishlistItem = BODL.wishlist.items.filter((i) => i.product_id === parseInt(BODL.getQueryParamValue('added_product_id'))).map((p) => ({
+      content_id: p.product_id,
+      // Commenting out: category data doesn't currently exist on wishlist items
+      // content_category: p.does_not_exist, 
+      content_name: p.name,
+      content_type: "product_group",
+      currency: p.price.without_tax.currency,
+      price: p.price.without_tax.value,
+      value: p.price.without_tax.value,
+    }));
+    
+    sampleAnalyticsProvider.instance('<%= property_id %>').track('AddToWishlist', addedWishlistItem[0]);
+  }
+</script>
+```
+
+## Order complete
+The following snippet uses the unauthenticated Storefront API to request the items from a completed order and concatenate them into a single array of physical items, digital items, and gift certificates.
+
+```handlebars title="Sample Script Code Start: Purchase Event" lineNumbers
+<script>
+  fetch('/api/storefront/order/' + BODL.order.id, {
```

Comment:
edited to use BODL to construct storefront API call rather than straight Stencil templating that would be likely to encounter parse errors.